plugins/user-authenticators/ldap/src/main/java/org/apache/cloudstack/ldap/LdapManagerImpl.java

```diff
@@ -432,14 +430,18 @@ private LinkDomainToLdapResponse linkDomainToLdap(Long domainId, String type, St
         LinkType linkType = LdapManager.LinkType.valueOf(type.toUpperCase());
         LdapTrustMapVO vo = _ldapTrustMapDao.persist(new LdapTrustMapVO(domainId, linkType, name, accountType, 0));
         DomainVO domain = domainDao.findById(vo.getDomainId());
+        String domainUuid = getDomainUuid(domain, vo);
+        return new LinkDomainToLdapResponse(domainUuid, vo.getType().toString(), vo.getName(), vo.getAccountType().ordinal());
+    }
+
+    private String getDomainUuid(DomainVO domain, LdapTrustMapVO vo) {
         String domainUuid = "<unknown>";
         if (domain == null) {
-            logger.error("no domain in database for id " + vo.getDomainId());
+            logger.error("no domain in database for id {}", vo.getDomainId());
```

Comment:
```suggestion
            logger.error("No domain in database for id {}", vo.getDomainId());
```
However, do we need to log this with the database ID?